utils/coord2area_def.py

```diff
@@ -104,19 +104,33 @@
         " +".join(("proj=" + proj + ",lat_0=" + str(lat_0) +
                    ",lon_0=" + str(lon_0) + ",ellps=WGS84").split(","))
 
-    print proj4_string
-
-    print "REGION:", name, "{"
-    print "\tNAME:\t", name
-    print "\tPCS_ID:\t", proj + "_" + str(lon_0) + "_" + str(lat_0)
-    print ("\tPCS_DEF:\tproj=" + proj +
-           ",lat_0=" + str(lat_0) +
-           ",lon_0=" + str(lon_0) +
-           ",ellps=WGS84")
-    print "\tXSIZE:\t", xsize
-    print "\tYSIZE:\t", ysize
-    print "\tAREA_EXTENT:\t", area_extent
-    print "};"
+    print(proj4_string)
+
+    print("REGION:", name, "{")
+    print("\tNAME:\t", name)
+    print("\tPCS_ID:\t", proj + "_" + str(lon_0) + "_" + str(lat_0))
+    print("\tPCS_DEF:\tproj=" + proj +
+          ",lat_0=" + str(lat_0) +
+          ",lon_0=" + str(lon_0) +
+          ",ellps=WGS84")
+    print("\tXSIZE:\t", xsize)
+    print("\tYSIZE:\t", ysize)
+    print("\tAREA_EXTENT:\t", area_extent)
+    print("};")
```

Comment:
I personally think we should just get rid of the old format.

Reply:
@djhoese Agreed!